### ISSUE TITLE
List of all Bugs & Things to add

### ISSUE THREAD

**Issue by YeahOMA** [ongoingotrich/Raven-issues-#12]
Bugs to Fix:

- Fix compatibility with clients 
- Modules are very buggy, sometimes they don't even work when they are turned on
- Polish client a little bit more, instead of adding more modules try to make the client bug free with the modules it has right now and then expand the modules list slowly.

If this is going to be a CrystalPVP client you need these modules:

- Strafe
- Fullbright / And other render modules
- Velocity
- Better AutoCrystal
- Utility Modules, see my last post

Basically make it have all the modules you could need/want in anarchy.

Apart from the bugs/needed modules it is a good client.
